### PR TITLE
doc: update website docs

### PR DESCRIPTION
**Pull request by b00f**
This PR updates some documents on website and fixes some mistakes.

### CODE REVIEW

File: content/protocol/consensus/solid-state-proof-of-stake.md [hunk 1]
```diff
@@ -3,40 +3,43 @@ title: Solid State Proof of Stake
...
-Some of these users volunteer to collect, validate, and record transactions,
-ensuring that everything runs smoothly.
+To understand how Proof of Stake works, think of a community bank run by its members without a central authority.
+The members decide to manage the bank together.
+Some members volunteer to collect, check, and record transactions to keep everything running smoothly.
 
-These volunteers, known as validators, must temporarily lock or freeze some of their money as a stake.
-This stake can't be transferred or used.
+These volunteers, known as validators, must temporarily lock up some of their money as a stake.
+This staked money can’t be moved or used.
```
Comment (Mr-HoDL58) [inline content/protocol/consensus/solid-state-proof-of-stake.md:25]: staked coins?

File: content/protocol/consensus/solid-state-proof-of-stake.md [hunk 2]
```diff
@@ -3,40 +3,43 @@ title: Solid State Proof of Stake
...
 
 From time to time, one of the validators is chosen by the others to collect all the recent transactions,
 bundle them together, and send a copy to the other validators.
 If a supermajority of the validators agree with the proposed bundle by signing it,
 the bundle will be committed to the bank's ledger.
 
-In this system, validators have no incentive to behave maliciously or dishonestly.
-If they do, they risk harming the bank's reputation and the value of their own stakes as well.
+In this system, validators are encouraged to act honestly.
+If they don’t, they risk damaging the bank’s reputation and losing the value of their staked money.
```
Comment (Mr-HoDL58) [inline content/protocol/consensus/solid-state-proof-of-stake.md:34]: stakes coins?

**Review by Mr-HoDL58 — APPROVED** [review]
money refernces changed to coins?

File: content/protocol/consensus/solid-state-proof-of-stake.md (hunk 1 above)
Comment (b00f) [reply content/protocol/consensus/solid-state-proof-of-stake.md:25]: Here we talk about an example of Community Bank to explain the PoS.

File: content/protocol/consensus/solid-state-proof-of-stake.md (hunk 2 above)
Comment (b00f) [reply content/protocol/consensus/solid-state-proof-of-stake.md:34]: Same example here.

### Community Bank Example

To understand how Proof of Stake works, think of a community bank run by its members without a central authority.
The members decide to manage the bank together.
Some members volunteer to collect, check, and record transactions to keep everything running smoothly.

These volunteers, known as validators, must temporarily lock up some of their money as a stake.
This staked money can’t be moved or used.
The more money a validator stakes, the more influence they have in the system.

From time to time, one of the validators is chosen by the others to collect all the recent transactions,
bundle them together, and send a copy to the other validators.
If a supermajority of the validators agree with the proposed bundle by signing it,
the bundle will be committed to the bank's ledger.

In this system, validators are encouraged to act honestly.
If they don’t, they risk damaging the bank’s reputation and losing the value of their staked money.

**Pull request merged**